lib/dumbo/function.rb

```diff
@@ -45,12 +45,16 @@ def load_attributes
           pg_catalog.pg_get_function_result(p.oid) as result_type,
           pg_catalog.pg_get_function_arguments(p.oid) as args,
           pg_catalog.pg_get_function_identity_arguments(p.oid) as arg_types,
-          CASE
-            WHEN p.proisagg THEN 'agg'
-            WHEN p.proiswindow THEN 'window'
-            WHEN p.prorettype = 'pg_catalog.trigger'::pg_catalog.regtype THEN 'trigger'
-            ELSE 'normal'
-          END as "type",
+           CASE p.prokind
```

Comment:
PG 11 made it a pain by renaming pro kind, since pg10 is still out there and supported until 2020 we might want to retain compatibility. I'd execute different SQLs depending on  whether `current_setting('server_version_num')::integer > 110000`

Reply:
I guess we should focus for now making dumbo usable for adjust.
Making it usable for a broader audience could be the next step.

Reply:
Agreed.